packages/@expo/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -41,6 +42,13 @@ export interface DefaultConfigOptions {
    * is subject to change, and native support for CSS Modules may be added in the future during a non-major SDK release.
    */
   isCSSEnabled?: boolean;
+
+  dangerous_beforeChunkSerialization?: (...params: SerializerParameters) => SerializerParameters;
```

Comment:
We use `unstable_` in the metro space. Also please make the type an array of serializer plugins:
```js
unstable_serializerPlugins: ((...params: SerializerParameters) => SerializerParameters)[]
```